tests/pyapi/test_cluster.py

```diff
@@ -1,5 +1,4 @@
 import pytest
-
```

Comment:
Do you have exactly the versions of `black`/`flake8` etc. as is used on CI? `tests/requirements.txt`. These formatting changes seem to be performed back-and-forth.

Reply:
```
Name: black
Version: 22.6.0
```

Reply:
CI uses `22.3.0`. Please (uninstall `black` and) reinstall tools according to `tests/requirements.txt`, thanks.

Reply:
Or update tests/requirements.txt on newer versions? (As this commit updates dependencies?)

Reply:
Fine by me :)